linode/nbs/framework_schema_datasource.go

```diff
@@ -0,0 +1,35 @@
+package nbs
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/linode/terraform-provider-linode/linode/helper/frameworkfilter"
+	"github.com/linode/terraform-provider-linode/linode/nb"
+)
+
+var filterConfig = frameworkfilter.Config{
```

Comment:
We probably can make the other attributes, i.e. `created`, be client filterable. Just configure them `APIFilterable: false` with appropriate `TypeFunc`.

Reply:
Good idea. I pushed up a change to include more attributes filterable (though `created` and `updated` won't work because of the way we parse time fields in linodego).